shopify_python/google_styleguide.py

```diff
@@ -228,3 +237,15 @@ def __use_simple_list_comp(self, node):  # type: (astroid.ListComp) -> None
         """List comprehensions are okay to use for simple cases."""
         if len(node.generators) > 1:
             self.add_message('complex-list-comp', node=node)
+
+    def __use_cond_expr(self, node):  # type: (astroid.If) -> None
+        """Only one liner conditional expressions"""
+
+        variables = []
+        for child in list(node.get_children()):
+            if str(child) == "Assign()":
```

Comment:
Use `isinstance` to ensure that this is an assignment instead of relying upon `str`